src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerEngine.java

```diff
@@ -291,24 +288,20 @@ private void initializeActiveRegionEvaluationGenotyperEngine() {
     }
 
     /**
-     * Create the default read filter for use with the HaplotypeCaller
-     *
-     * @param hcArgs HaplotypeCaller arguments
-     * @param header reads header
-     * @return the default read filter for use with the HaplotypeCaller
+     * @return the default set of read filters for use with the HaplotypeCaller
      */
-    public static CountingReadFilter makeStandardHCReadFilter( final HaplotypeCallerArgumentCollection hcArgs, final SAMFileHeader header ) {
-        Utils.nonNull(hcArgs);
-        Utils.nonNull(header);
-
-        return new CountingReadFilter("MAPPING_QUALITY", new MappingQualityReadFilter(hcArgs.MIN_MAPPING_QUALITY_SCORE))
-                .and(new CountingReadFilter("MAPPING_QUALITY_AVAILABLE", ReadFilterLibrary.MAPPING_QUALITY_AVAILABLE))
-                .and(new CountingReadFilter("MAPPED", ReadFilterLibrary.MAPPED))
-                .and(new CountingReadFilter("PRIMARY_ALIGNMENT", ReadFilterLibrary.PRIMARY_ALIGNMENT))
-                .and(new CountingReadFilter("NOT_DUPLICATE", ReadFilterLibrary.NOT_DUPLICATE))
-                .and(new CountingReadFilter("PASSES_VENDOR_QUALITY_CHECK", ReadFilterLibrary.PASSES_VENDOR_QUALITY_CHECK))
-                .and(new CountingReadFilter("GOOD_CIGAR", ReadFilterLibrary.GOOD_CIGAR))
-                .and(new CountingReadFilter("WELLFORMED", new WellformedReadFilter(header)));
+    public static List<ReadFilter> makeStandardHCReadFilters() {
+        List<ReadFilter> filters = new ArrayList<>();
+        filters.add(new MappingQualityReadFilter(READ_QUALITY_FILTER_THRESHOLD));
+        filters.add(ReadFilterLibrary.MAPPING_QUALITY_AVAILABLE);
+        filters.add(ReadFilterLibrary.MAPPED);
```

Comment:
These three filters are very cheap compared to the later filters (like `WellformedReadFilter`), so I think we're good for now in terms of short-circuiting, unless something shows up on the profiler.